jetty-client/src/main/java/org/eclipse/jetty/client/GZIPContentDecoder.java

```diff
@@ -18,26 +18,13 @@
 
 package org.eclipse.jetty.client;
 
-import java.nio.ByteBuffer;
-import java.util.Arrays;
-import java.util.zip.DataFormatException;
-import java.util.zip.Inflater;
-import java.util.zip.ZipException;
-
-import org.eclipse.jetty.util.BufferUtil;
 
 /**
  * {@link ContentDecoder} for the "gzip" encoding.
+ * 
  */
-public class GZIPContentDecoder implements ContentDecoder
+public class GZIPContentDecoder extends org.eclipse.jetty.http.GZIPContentDecoder implements ContentDecoder
```

Comment:
This now uses the common implementation from HTTP indirectly.   Perhaps the client could be changed to use it directly?